client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -47,12 +55,110 @@ define([
                     return posts;
                 });
         }
+        function retrieveItems(blog_id, items_criteria) {
+            return api('blogs/<regex(\"[a-f0-9]{24}\"):blog_id>/items', {_id: blog_id})
+                .query(items_criteria)
+                .then(function(data) {
+                    return data._items;
+                });
+        }
+        function updateLastIndex(blog) {
```

Comment:
Hard for me to understand the meaning of this function.
Why this function require a blog as parameter ?
What are the consequences if we call this function ?
It should be more explicit because this method is accessible from the service's API